src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -362,6 +363,7 @@ typedef struct {
     size_t rx_iov_limit;
     size_t rma_iov_limit;
     int max_ch4_vnis;
+    int max_rma_tx_cnt;         /* Max number of transmit context on one RMA scalable EP */
     size_t max_order_raw;
```

Comment:
`max_rma_tx_cnt ` is used only for scalable EP, can we change the parameter name to something like `max_rma_sep_tx_cnt` so it is clear ?